scalableminds/webknossos
# Beautify right menu

This PR adds some css changes to beautify the look of tracing views
- https://beautifyrightmenu.webknossos.xyz

### Steps to test:
- open each tracing view
- in each tracing view there should be the following differences to the master:
- all info-tabs have a padding in each direction so that they do not touch the grey borders of a layout container directly
- all headers of the containers should have a slightly higher height, so that they do not directly stick to their container at their bottom
- all grey borders of the layout-containers should be still and always visible
- the behavior of the layout should still be the same

- as these are only css changes please that them on all installed browsers (I tested them in firefox and chrome)

### additionally:
I do not think we need a changelog entry here, because this is only a slight visual tweak.
### Issues:
- fixes #3292 

------
- ~~~[ ] Updated [changelog](../blob/master/CHANGELOG.md#unreleased)~~~
- ~~~[ ] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable~~~
- ~~~[ ] Updated [documentation](../blob/master/docs) if applicable~~~
-  ~~| ] Needs datastore update after deployment~~~
